src/MudDemo/MudDemo.Client/Models/Article/ArticlePreviewModel.cs

```diff
@@ -1,11 +1,16 @@
+using Newtonsoft.Json;
```

Comment:
Maybe we don't need to specify JsonProperty so we don't need to add Newtonsoft package? As we don't serialize